book/src/SUMMARY.md

```diff
@@ -26,3 +26,5 @@ SPDX-License-Identifier: MIT OR Apache-2.0
   - [Shared types](./bridge/shared_types.md)
   - [Attributes](./bridge/attributes.md)
   - [Traits](./bridge/traits.md)
+- [For Contributors: CXX-Qt Internals](./internals/index.md)
+  - [Build System](./internals/build-system.md)
```

Comment:
could you add an entry into the changelog to detail that you have changed the build system as we forgot on the other changes

Reply:
Ah, good point!

Reply:
Done